prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+  The storage disk drive(s) where the data for the defined or used shares resources
+  wil be stored. **Cannot be the same of the system drive(s) and sizes will
+  depends of the usage of the resources**. Shared resources are managed over
+  partitions, which is discussed in the software requirements topic below.
+  The recommendation for the Data drives must be Solid State Disks (SSD [2]_ disks)
+  for best performance or spinning Hard Disk Drives (HDD).
+
+The memory (RAM)
+^^^^^^^^^^^^^^^^
+
+Enough RAM is vital to maintaining peak performance. There are several combinations
+of installation sizes but the **recommendation is at least 4Gib for novice administrators**.
+
+For best practice you should have 8GiB of RAM for basic operations in default installation.
+and no matter the size, RAM must be configured in Dual Channel mode [8]_ 
+inclusivelly if there are little amount of, it will improve performance.
+
+Unless other NAS systems the |omv| system can run in at least 1Gib of RAM, but
+of course will need expertise.
+
+The communication card (NIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A NAS system as means is a network oriented system, of course |omv| is the
+most flexible NAS system in the world and allows multiple ways of communication, 
+this is becouse |omv| offers several modes of shared resources like RSYNC that 
+do not depends on networking. But there are few key points:
+
+- The NAS write speed to storage is limited to the speed of your NAS computer NIC card
+- The NAS write speed to storage is limited to the speed of your network
+- Speed of writing to the storage depends on ratio of the number of users accessing it.
+
+Full-duplex NICs are recommended, half-duplex NICs typically offer 10/100 Mbit/s
+and storage drives far exceed those speeds but are of course supported by |omv|.
+
+Any supported NIC vendor is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+The architecture (CPU)
+^^^^^^^^^^^^^^^^^^^^^^
+
+We have already denoted in the requirements table the supported architectures or
+processors (which reduces and determines which computers can be used) but this is
+really a reference since it is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+Any computer system supported by the common requirements of Debian operating
+system could be an |omv| system installation target. **Currently we supports
+amd64 (x66 64bits), i386 (x86 32bits), ARMv7+ARMv8 (arm64/armel/armhf)**.
+
+Software requirements
+---------------------
+
+======  =================  ==============  ==============  =======================================
+ Item    Software           Minimal         Best            Recommendation
+======  =================  ==============  ==============  =======================================
+ OS      Debian Linux       oldoldstable    stable          Current stable (plus 1 month released)
```

Comment:
This is incorrect. Every OMV release is tied to a specific Debian version. You can link to https://github.com/openmediavault/openmediavault-docs/blob/master/releases.rst here.